gammapy/maps/axes.py

```diff
@@ -1122,6 +1122,10 @@ def downsample(self, factor):
         ----------
         factor : int
             Downsampling factor.
+        strict : bool
+
+            If True, then the remainder is placed in the last bin
```

Comment:
The current docstring `requires nbin to be dividable by the factor..`, must be adapted.
I guess the behaviour should be
- `strict=True` : nbin must to be divisible by the factor 
- `strict=False` the reminder bins are put in the last bin of the new axis